code/nodes/dataCheck/utils.q

```diff
@@ -187,30 +221,38 @@ dataCheck.i.delFiles:{[config;ignore;mainFileExists;loggingExists]
 
 // @kind function
 // @category dataCheckUtility
-// @fileoverview If savePath and logging already exist, give warning or error out depening on
-//  ignoreWarning option
-// @param config {dict} Configuration information assigned by the user and related to the current run
+// @desc If savePath and logging already exist, give warning or 
+//  error out depening on ignoreWarning option
+// @param config {dictionary} Configuration information assigned by the user
+//   and related to the current run
 // @param ignore {int}  The utils.ignoreWarnings options set i.e. 0, 1 or 2
-// @param mainFileExists {bool} Whether the savePath exists if saveOption is greater than 0
-// @param loggingExists {bool} Whether the logging path exists if logging option is chosen
-// @return {null;err} Error if logfile or savePath already exists or give warning
+// @param mainFileExists {boolean} Whether the savePath exists if saveOption 
+//   is greater than 0
+// @param loggingExists {boolean} Whether the logging path exists if logging
+//   option is chosen
+// @return {::|err} Error if logfile or savePath already exists or give warning
 dataCheck.i.printWarning:{[config;ignore;mainFileExists;loggingExists]
   if[ignore=0;:()];
   index:$[ignore=2;0;1];
   if[mainFileExists;
-      dataCheck.i.warningOption[config;ignore] utils.printWarnings[`savePathExists]index];
+      dataCheck.i.warningOption[config;ignore] 
```

Comment:
Should be inline with the last `]`.
So - double indentation from start of if statement. See above comment for example